content/en/dashboards/guide/how-to-use-terraform-to-restrict-dashboard-edit.md

```diff
@@ -7,36 +7,59 @@ aliases:
+The `restricted_roles` attribute can be used to restrict editing of the dashboard to specific roles. The field takes a list of IDs of roles, and authorizes any associated users.
 
-## Restricting a dashboard
+Example usage:
 
-If you're already using `is_read_only` in your definition, this continues to work while your organizational users see and use the role list. To properly synchronize your Terraform definitions and the experience the Datadog application, complete the following steps:
+```hcl
+resource "datadog_dashboard" "example" {
+  title         = "Example dashboard"
+  restricted_roles = ["<role_id_1>", "<role_id_2>"]
+}
+```
 
-1. Update your Datadog Terraform provider to version 3.1.0 or above.
+<div class="alert alert-warning">
+The `is_read_only` attribute is deprecated. It is recommended to use the `restricted_roles` attribute or Restriction Policies to manage access to your dashboards.
+</div>
 
-2. Find the UUID of the roles that you want to restrict by either retrieving the UUID from the [Roles APIs][2] or Roles UI, or referring to role ID as defined in Terraform for [Terraform managed Roles][3].
+## Restricting a dashboard using a restriction policy
 
-3. Where you're using `is_read_only` in Dashboard definitions, replace them with `restricted_roles`:
+{{< callout url="https://www.datadoghq.com/support/" >}}
+Restriction policies are in beta. Reach out to Datadog support to enable the feature.
+{{< /callout >}} 
 
-{{< img src="dashboards/guide/terraform_is_read_only_definition.png" alt="Read-only dashboards" style="width:80%;">}}
+[Restriction Policies][1] allow restricting resources to various principals including Roles, Teams, Users, and Service Accounts.
 
-{{< img src="dashboards/guide/terraform_restricted_role_definition.png" alt="Dashboards restricted by roles" style="width:80%;">}}
+Example usage:
 
-## Common issues
+```hcl
+resource "datadog_dashboard" "example" {
+  title         = "Example dashboard"
+  # Do not use restricted_roles or is_read_only attributes
+}
 
-### `is_read_only` is still enabled for my dashboard
+resource "datadog_restriction_policy" "example" {
+ resource_id = "dashboard:${datadog_dashboard.example.id}"
+  bindings {
+     principals = ["org:<org_id>"]
+     relation = "viewer"
+  }
+  bindings {
+     principals = ["role:<role_id_1>", "role:<role_id_2>"]
+     relation = "editor"
+  }
+}
+```
 
-This configuration still works. Every Terraform run detects any changes to roles or read only and notify you if `is_read_only` is being changed.
+Role IDs can be retrieved from the [Roles API][2], [Roles UI][5], or by using the role ID defined in Terraform for [datadog_role][3] resources.
 
-### Terraform is warning that `is_read_only` has been removed
+Org ID can be obtained from the [GET /api/v2/org API][4].
```

Comment:
Good catch - that was because your user doesn't have org_management permission. Which isn't something we'd expect from every doc reader. So, switched to another undocumented API: https://app.datadoghq.com/api/v2/current_user (it's pretty easy to find org ID in there too)